dashboard/app/controllers/api/v1/pd/workshop_survey_report_controller.rb

```diff
@@ -104,6 +110,7 @@ def local_workshop_survey_report
       end
     end
 
+    # TODO: this action is to be deprecated
     # GET /api/v1/pd/workshops/:id/local_workshop_daily_survey_report
     def local_workshop_daily_survey_report
```

Comment:
If we're actually deprecating this now (we don't expect anyone to call it anymore) let's add a `Honeybadger.notify` call in this controller method so we can see whether anyplace is still calling it.

Reply:
My bad. What I mean is client shouldn't hit this action directly anymore. It becomes a private helper function now.

Reply:
I've searched our code base to make sure there is no reference of `local_workshop_daily_survey_report` outside of this file.